actions/checkout_android/action.yml

```diff
@@ -0,0 +1,16 @@
+name: 'Setup for all'
+description: "Setup shared for all jobs"
+runs:
+  using: 'composite'
+  steps:
+      - name: Checkout Android
+        uses: actions/checkout@v4
+        with:
+          repository: 'oxen-io/session-android'
+          path: 'android'
+          submodules: recursive
+          ref: 'release/1.20.0'
```

Comment:
Should this `ref` be removed?

Reply:
not for now, otherwise we don't target the right branch for session-android. We will once that branch is merged to dev